shared/menubar/space-warning.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../common-adapters'
+
+export type BannerType = 'warning' | 'failure' | 'none'
```

Comment:
Perhaps we can just use `Types.MainBannerType` here (and maybe rename it into something more generic like `Types.BannerType`)

Reply:
That's a different thing. MainBannerType is offline vs out of space - this is out of space vs low on space.

Reply:
I see. Makes sense. Thanks!

Not suggesting a change here but I'm thinking perhaps we can just have the BannerType cover all them, and in Files or menubar, just don't show anything if it's not a banner we want to show (e.g. low space in main window). Not sure if it'd actually be better though since we have only these banners. So really just thinking out loud.

Reply:
I feel like it would be annoying to have it be all one type, since there can be different banners in different places, if e.g. you're offline and have >1GB of space available.